index.html

```diff
@@ -82,7 +82,38 @@ <h1 class="worktext">My Recent Works</h1>
                <li> <a href="#">Get My Resume</a></li>
             </ul>
         </div>
-       
+        <footer>
+            <p class="context">I'm always interested in hearing about new projects, 
+                so if you'd like to chat please get in touch.</p>
+            <form action="https://formspree.io/f/xwkzpzlo" id="contactform" method="post">
+                <ul id="listdetails">
+                   <li id="namefield">
+                    <!-- <label for="name" id="namelabel">Full name</label> -->
```

Comment:
- [ ] Kindly remove the comments of non-used code. 🤓

Reply:
This is well noted @GiuseppeTG .
Let me do that right away.